build/devenv/environment.go

```diff
@@ -283,153 +285,195 @@ func NewEnvironment() (in *Cfg, err error) {
 			return nil, err
 		}
 	}
+	///////////////////////////
+	// END: Deploy contracts //
+	///////////////////////////
+
+	////////////////////////////
+	// Start: Launch CL Nodes //
+	////////////////////////////
```

Comment:
These cheesy block comments can go away in the future. For now I've been getting lost in this file and they've been helpful.